audio/audacity/Portfile

```
@@ -185,7 +185,8 @@ if {[variant_isset wxsystem]} {
 
 depends_build-append \
                     port:pkgconfig \
-                    port:python37
+                    port:python37 \
+                    port:cctools
 set python_bin      ${prefix}/bin/python3.7
```

Python 3.10 is preferred today. Can that version be used instead?
```suggestion
                    port:python310 \
                    port:cctools
set python_bin      ${prefix}/bin/python3.10
```